src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,29 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
make `const { name, email, body } = comment` to get rid of `comment.smth` just `smth`